src/Compilers/CSharp/Portable/Errors/ErrorCode.cs

```diff
@@ -1273,7 +1273,9 @@ internal enum ErrorCode
         ERR_BadPdbData = 8036,
         // available 8037-8039
         INF_UnableToLoadSomeTypesInAnalyzer = 8040,
-        // available 8041-8049
+        WRN_LoadedAnalyzerDiffers = 8041,
+        WRN_MissingAnalyzerDependency = 8042,
```

Comment:
I am not sure. Allowing that via /nowarn /warnaserror seems fine if users really want to do that. But it feels strange that we would advertise these warnings in solution explorer / ruleset editor.

@mavasani I don't feel very strongly about this. I was asking mainly because I didn't know whether we do anything to make other warnings for analyzer loading / running (that fall in a similar class) not-configurable and whether we should do the same thing for these new warnings for consistency. But sounds like we don't do anything special for any such warnings - so I am fine with leaving things as is.